doc/cabal-package-description-file.rst

```diff
@@ -2133,6 +2133,10 @@ system-dependent values for these fields.
     supported libraries. Depending on your system you may need to adjust
     ``PKG_CONFIG_PATH``.
 
+.. pkg-field:: private-build-depends: private scope
+
+    `private-build-depends` introduces a new private scope,... TODO
```

Comment:
todo

Reply:
I've added the documentation to the user's guide under private dependencies and expanded this field's explanation.